java/maven.embedder/external/apache-maven-3.9.6-failureaccess-1.0.1-license.txt

```diff
@@ -1,9 +1,9 @@
-Name: ValidationAPI
-Description: Provides utilities for adding input validation to Swing forms quickly and easily
-Version: 1.14.1
-Origin: GitHub
+Name: Guava - Failure Access Library
```

Comment:
For Guava and failureaccess inside the Maven distribution, I'd propose to specify the license explicitly. It is Apache 2.0 License, of course, but having the license files allows us to specify metadata - like version, which then can be used to build dependency information, for example.